docs/guides/data-quering.mdx

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 3
+title: Data Querying
+description: Complete guide on how to query data from ReductStore using the CLI, HTTP API, and SDKs.
+---
+
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+    <link rel="canonical" href="https://www.reduct.store/docs/guides/data-quering"/>
+</head>
+
+# Data Querying
+
+## Concepts
+
+ReducStore provides an efficient data retrieval solution by batching multiple records within a specified time interval into a single HTTP request, which is beneficial for managing large volumes of data as it reduces the number of requests and overall delay.
+
+The query process is designed as an iterator, returning a batch of records in each iteration. This method allows data to be processed in segments, an approach that is useful when managing large datasets.
+
+While it is possible to retrieve a record by its timestamp, this method is less optimal than querying by a time range due to the lack of batching. However, this approach can be used when timestamps serve as identifiers for non-time series records, such as models or configurations.
```

Comment:
suggestion: "However, this approach can be useful for querying specific versions of non-time series records, such as AI models, configurations, or file versions, when timestamps are used as identifiers."